dashboard/src/components/AipDeletionReviewAlert.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { useAipStore } from "@/stores/aip";
+
+const aipStore = useAipStore();
+
+const { note } = defineProps<{
+  note: string;
+}>();
+
+const body = computed(() => note);
```

Comment:
Is this necessary? Can't you just call `{{ note }}` directly in the template?

Reply:
That's what I tried initially and it din't work, not sure what I did wrong, it works now ;)